test/adapters/optimisticAuctionRebalanceExtension.spec.ts

```diff
@@ -0,0 +1,757 @@
+              await setV2Setup.auctionModule.rebalanceInfo(setToken.address)
+            ).raiseTargetPercentage;
+
+            expect(actualRaiseTargetPercentage).to.eq(subjectRaiseTargetPercentage);
+          });
+
+          describe("when the caller is not the operator", async () => {
+            beforeEach(async () => {
+              subjectCaller = await getRandomAccount();
+            });
+
+            it("should revert", async () => {
+              await expect(subject()).to.be.revertedWith("Must be operator");
+            });
+          });
+        });
+
+        describe("#setBidderStatus", () => {
+          let subjectBidders: Address[];
+          let subjectStatuses: boolean[];
+          let subjectCaller: Account;
+
+          beforeEach(async () => {
+            subjectBidders = [methodologist.address];
+            subjectStatuses = [true];
+            subjectCaller = operator;
+          });
+
+          async function subject(): Promise<ContractTransaction> {
+            return await auctionRebalanceExtension
+              .connect(subjectCaller.wallet)
+              .setBidderStatus( subjectBidders, subjectStatuses);
+          }
+
+          it("should correctly set the bidder status", async () => {
+            await subject();
+
+            const isCaller = await setV2Setup.auctionModule.isAllowedBidder(
+              setToken.address,
+              subjectBidders[0],
+            );
+
+            expect(isCaller).to.be.true;
+          });
+
+          describe("when the caller is not the operator", () => {
+            beforeEach(async () => {
+              subjectCaller = await getRandomAccount();
+            });
+
+            it("should revert", async () => {
+              await expect(subject()).to.be.revertedWith("Must be operator");
+            });
+          });
+        });
+
+        describe("#setAnyoneBid", () => {
+          let subjectStatus: boolean;
+          let subjectCaller: Account;
+
+          beforeEach(async () => {
+            subjectStatus = true;
+            subjectCaller = operator;
+          });
+
+          async function subject(): Promise<ContractTransaction> {
+            return await auctionRebalanceExtension
+              .connect(subjectCaller.wallet)
+              .setAnyoneBid(subjectStatus);
+          }
+
+          it("should correctly set anyone bid", async () => {
+            await subject();
+
+            const anyoneBid = await setV2Setup.auctionModule.permissionInfo(setToken.address);
+
+            expect(anyoneBid).to.be.true;
+          });
+
+          describe("when the caller is not the operator", () => {
+            beforeEach(async () => {
+              subjectCaller = await getRandomAccount();
+            });
+
+            it("should revert", async () => {
+              await expect(subject()).to.be.revertedWith("Must be operator");
+            });
+          });
+        });
+
+        describe("#removeAdapter", () => {
```

Comment:
let's remove this test from this file since its a BaseManager function

Reply:
Done